lib/manageiq/providers/ibm_cloud/cloud_tools.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'logging'
 require 'ibm_vpc'
+require 'ibm_cloud_activity_tracker'
```

Comment:
Travis is failing with: 
```
Failure/Error: require 'ibm_cloud_activity_tracker'
LoadError:
  cannot load such file -- ibm_cloud_activity_tracker
# ./lib/manageiq/providers/ibm_cloud/cloud_tools.rb:5:in `<top (required)>'
# ./spec/models/manageiq/providers/ibm_cloud/vpc/cloud_tools_spec.rb:3:in `<top (required)>'
```

@agrare Do I need to require the gem here if its already listed as a dependency in the gemspec?